### ISSUE TITLE
--root-path not working

### ISSUE THREAD

**Issue by z9xx7o0** [zotify-dev/zotify#94]
I've set up the program on termux and want  to change the download path so that it is easily accessible 

zotify --root-path /data/data/com.termux/files/home/storage/music/

It didn't change the path. Instead, it gave me a search option.

I tried changing ROOT_PATH from the config file, but that broke the entire program, and I had to re install termux, python3, ffmpeg, git, pip, pipx and zotify

I also tried --output, but it gave me the same result.

**Comment by shinji257** [comment]
Did you try the above and specify a track/album/artist to download on the command line?  For me all of the above works fine.

**Comment by z9xx7o0** [comment]
> Did you try the above and specify a track/album/artist to download on the command line?  For me all of the above works fine.

Yes, I did. I tried everything. The only other reason may be that I am not writing it correctly, is this how it should be written?

zotify --root-path /data/data/com.termux/files/home/storage/music/

**Comment by z9xx7o0** [comment]
> Did you try the above and specify a track/album/artist to download on the command line?  For me all of the above works fine.

Oh, after specifying a track, it worked. Is there a way to make that directory the permanent path for all downloads? And why is 	--skip-previously-downloaded not working? Am I using it wrong?

zotify 	--skip-previously-downloaded=True

**Comment by shinji257** [comment]
Edit the config and you should be able to set it there.  The song archive that skip-previously-downloaded is only generated when that option is enabled.  You can also have it skip ones that already exist.  If you need to create a fresh .song_archive file you can do so by combining all of the .song_id files that are scattered around.    Something like this worked for me.

`find . -name '.song_ids' -exec cat {} \; > .song_archive`

If the config isn't working then feel free to post it here and I'll look.

**Comment by farmaher** [comment]
I have the same issue here. I want to set a permanent path in config.json file. I put the following strings but this will break the functionality of the program completely:

    "ROOT_PATH": "E:\Spotify",
    "ROOT_PODCAST_PATH": "E:\Spotify\Podcasts",

I am not sure if I have missed something here.

**Comment by shinji257** [comment]
With Python on windows you either use \\ or / for path separation. \ all by
itself has special meaning in Python and you have to escape it if you are
going to use that. / on the other hand works regardless of the platform.
